src/EventListener/LogRevisionsListener.php

```diff
@@ -200,7 +215,7 @@ public function postPersist(LifecycleEventArgs $eventArgs): void
         // onFlush was executed before, everything already initialized
         $entity = $eventArgs->getEntity();
 
-        $class = $this->em->getClassMetadata(\get_class($entity));
+        $class = $this->em->getClassMetadata(get_class($entity));
```

Comment:
add `use function` statement for function `get_class` or revert back backslash

Reply:
Fixed